### PR TITLE
June release notes

### PR DESCRIPTION
**Pull request by dannify**

### CODE REVIEW

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+# June 26, 2023 Release
+
+Our summer break release is delivering new features, a new version of React Aria Components and support for React Strict Mode! React Spectrum and React Aria can now be run in React Strict Mode which provides better developer feedback for errors and warnings. Another React feature we now support is the built-in React `useId` in React 18 in place of our existing `useId` hook. See our [release notes below](#useid-changes) for more details on this feature.  Calendar and DatePicker have been updated to inclued additional options for controling the page behavior and locale overrides. We have also
+react strict mode
+pageBehavior
+useId
+
+
+## Enhancements
+* Finalize React strict mode issues - now fully supported - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4564)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:27]:
```suggestion
* Fix remaining React strict mode issues - now fully supported - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4564)
```

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+* Use React.useId if available - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4347)
+* Add `pageBehavior` prop to `Calendar` to control page duration - [@confix](https://github.com/confix) - [PR](https://github.com/adobe/react-spectrum/pull/4587)
+* Add `pageBehavior` to Spectrum `Calendar` and `RangeCalendar` - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4650)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:30]: probably don't need this in here twice?

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+## Fixes
+* Ensure that `useObjectRef` only calls function refs once in strict mode - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4586)
+* Ensure scrollable is set before onTouchMove - [@bbutler522](https://github.com/bbutler522) - [PR](https://github.com/adobe/react-spectrum/pull/3749)
+* ContextualHelp button should be labeled by the label in `Field` - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/4319)
+* `isLoading` state should include "Loading..." in the description in `Picker` - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/4109)
+* useLabels/useLabel consider reordering IDREFs so that aria-label comes before aria-labelledby - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/4191)
+* Update weekStartData.ts - [@rvramesh](https://github.com/rvramesh) - [PR](https://github.com/adobe/react-spectrum/pull/4598)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:48]: Maybe add a bit more details

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+* Fix `SSRProvider` return type - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4627)
+* Add a11y info for `Checkbox` `validationState` - [@jnurthen](https://github.com/jnurthen) - [PR](https://github.com/adobe/react-spectrum/pull/4003)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:50]: Should be under the docs section

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+* Improve DatePicker support with screen reader (NVDA) - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/4639)
+* Improve Firefox `Popover` synthetic blur on disabled button - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/4670)
+
+## Docs
+* Disable hidden select if select is disabled in `useSelect` example - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4552)
+* Fix broken `Collections` link in `Breadcrumbs` React Aria Components page - [@abdmmar](https://github.com/abdmmar) - [PR](https://github.com/adobe/react-spectrum/pull/4569)
+* Update broken link in `@internationalized` date readme - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/4596)
+* Fix scroll position jump on mobile for column resize examples in `TableView` - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4646)
+* refac: Remove unused variables from examples in `useTabList` - [@ryo-manba](https://github.com/ryo-manba) - [PR](https://github.com/adobe/react-spectrum/pull/4690)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx]: what is refac?

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+* Add `pageBehavior` to RAC/Spectrum Calendar and RangeCalendar - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4650)
+* RAC `DropZone` - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/4451)
+* Add TagGroup component to RAC - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4661)
+* Add WHCM styles for RAC `TagGroup` docs - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4680)
+* Fix file upload dialog in RAC `DropZone` - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/4681)
+* Update `pageBehavior` docs to separate examples - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4682)
+* Make RAC overlay open props override `DialogTrigger` state - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4684)
+
+**React Aria Components Tabs API changes**
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:88]:
```suggestion
#### React Aria Components Tabs API changes
```

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+
+## Under construction
+
+Fixes to pre-released components are listed below. Please feel free to try them out, and report any issues you encounter.
+
+### React Aria Components (Alpha)
+* Use span for VisuallyHidden when inside a `label` - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4590)
+* Refactor React Aria Components Tabs API - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4540)
+* Fix keyboardActivation propagation and remove `disabledKeys` from React Aria Components `TabList` props - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4608)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:76]: can remove this, it was a bug fix for the previous refactor

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+* Add support for standalone RAC popovers - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4656)
+* Add `pageBehavior` to RAC/Spectrum Calendar and RangeCalendar - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4650)
+* RAC `DropZone` - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/4451)
+* Add TagGroup component to RAC - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4661)
+* Add WHCM styles for RAC `TagGroup` docs - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4680)
+* Fix file upload dialog in RAC `DropZone` - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/4681)
+* Update `pageBehavior` docs to separate examples - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4682)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:85]: fix for a previous item

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+* Add AnimatedCalendar example with framer-motion - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4623)
+* Update @types/react version, and fix forwardRef type with render props - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4600)
+* Add support for standalone RAC popovers - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4656)
+* Add `pageBehavior` to RAC/Spectrum Calendar and RangeCalendar - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4650)
+* RAC `DropZone` - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/4451)
+* Add TagGroup component to RAC - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4661)
+* Add WHCM styles for RAC `TagGroup` docs - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4680)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx]: fix for previous

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+* Make RAC overlay open props override `DialogTrigger` state - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4684)
+
+**React Aria Components Tabs API changes**
+
+In this release, we updated the [Tabs](https://react-spectrum.adobe.com/react-aria/Tabs.html) API in React Aria Components. The `<TabPanels>` component is no longer necessary and has been removed. Instead, each individual `<TabPanel>` now chooses whether to render itself based on the selected tab state. In addition, a new `shouldForceMount` prop has been added to force even non-selected tabs to render to the DOM. This can be used to implement animations between tabs or preserve state when a tab is not visible. Check out the [updated documentation](https://react-spectrum.adobe.com/react-aria/Tabs.html) for more details.
+
+## Released packages
+
+```
+
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:95]:
```suggestion
 - @adobe/react-spectrum@3.28.0
 - @internationalized/date@3.3.0
 - @internationalized/message@3.1.1
 - @internationalized/number@3.2.1
 - @internationalized/string-compiler@3.2.1
 - @internationalized/string@3.1.1
 - @react-aria/accordion@3.0.0-alpha.19
 - @react-aria/actiongroup@3.6.0
 - @react-aria/aria-modal-polyfill@3.7.3
 - @react-aria/autocomplete@3.0.0-alpha.18
 - @react-aria/breadcrumbs@3.5.3
 - @react-aria/button@3.8.0
 - @react-aria/calendar@3.4.0
 - @react-aria/checkbox@3.9.2
 - @react-aria/color@3.0.0-beta.22
 - @react-aria/combobox@3.6.2
 - @react-aria/datepicker@3.5.0
 - @react-aria/dialog@3.5.3
 - @react-aria/dnd@3.3.0
 - @react-aria/focus@3.13.0
 - @react-aria/grid@3.8.0
 - @react-aria/gridlist@3.5.0
 - @react-aria/i18n@3.8.0
 - @react-aria/interactions@3.16.0
 - @react-aria/label@3.6.0
 - @react-aria/landmark@3.0.0-beta.2
 - @react-aria/link@3.5.2
 - @react-aria/listbox@3.10.0
 - @react-aria/live-announcer@3.3.1
 - @react-aria/menu@3.10.0
 - @react-aria/meter@3.4.3
 - @react-aria/numberfield@3.6.0
 - @react-aria/overlays@3.15.0
 - @react-aria/progress@3.4.3
 - @react-aria/radio@3.6.2
 - @react-aria/searchfield@3.5.3
 - @react-aria/select@3.11.0
 - @react-aria/selection@3.16.0
 - @react-aria/separator@3.3.3
 - @react-aria/slider@3.5.0
 - @react-aria/spinbutton@3.5.0
 - @react-aria/ssr@3.7.0
 - @react-aria/switch@3.5.2
 - @react-aria/table@3.10.0
 - @react-aria/tabs@3.6.1
 - @react-aria/tag@3.1.0
 - @react-aria/textfield@3.10.0
 - @react-aria/toast@3.0.0-beta.2
 - @react-aria/toggle@3.6.2
 - @react-aria/tooltip@3.6.0
 - @react-aria/utils@3.18.0
 - @react-aria/virtualizer@3.9.0
 - @react-aria/visually-hidden@3.8.2
 - @react-spectrum/accordion@3.0.0-alpha.21
 - @react-spectrum/actionbar@3.1.0
 - @react-spectrum/actiongroup@3.9.0
 - @react-spectrum/autocomplete@3.0.0-alpha.19
 - @react-spectrum/avatar@3.0.2
 - @react-spectrum/badge@3.1.3
 - @react-spectrum/breadcrumbs@3.8.0
 - @react-spectrum/button@3.12.3
 - @react-spectrum/buttongroup@3.6.3
 - @react-spectrum/calendar@3.3.0
 - @react-spectrum/card@3.0.0-alpha.20
 - @react-spectrum/checkbox@3.7.3
 - @react-spectrum/color@3.0.0-beta.23
 - @react-spectrum/combobox@3.9.0
 - @react-spectrum/contextualhelp@3.6.0
 - @react-spectrum/datepicker@3.6.0
 - @react-spectrum/dialog@3.8.0
 - @react-spectrum/divider@3.5.3
 - @react-spectrum/dnd@3.3.0
 - @react-spectrum/form@3.6.3
 - @react-spectrum/icon@3.7.3
 - @react-spectrum/illustratedmessage@3.4.3
 - @react-spectrum/image@3.4.3
 - @react-spectrum/inlinealert@3.0.0-rc.0
 - @react-spectrum/label@3.13.0
 - @react-spectrum/labeledvalue@3.1.3
 - @react-spectrum/layout@3.5.3
 - @react-spectrum/link@3.5.3
 - @react-spectrum/list@3.6.0
 - @react-spectrum/listbox@3.11.0
 - @react-spectrum/menu@3.12.0
 - @react-spectrum/meter@3.4.3
 - @react-spectrum/numberfield@3.6.3
 - @react-spectrum/overlays@5.2.0
 - @react-spectrum/picker@3.11.0
 - @react-spectrum/progress@3.6.0
 - @react-spectrum/provider@3.8.0
 - @react-spectrum/radio@3.5.3
 - @react-spectrum/searchfield@3.7.3
 - @react-spectrum/slider@3.5.3
 - @react-spectrum/statuslight@3.5.3
 - @react-spectrum/switch@3.4.3
 - @react-spectrum/table@3.10.0
 - @react-spectrum/tabs@3.7.0
 - @react-spectrum/tag@3.1.0
 - @react-spectrum/text@3.4.3
 - @react-spectrum/textfield@3.10.3
 - @react-spectrum/theme-dark@3.5.3
 - @react-spectrum/theme-default@3.5.3
 - @react-spectrum/theme-express@3.0.0-alpha.5
 - @react-spectrum/theme-light@3.4.3
 - @react-spectrum/toast@3.0.0-beta.2
 - @react-spectrum/tooltip@3.5.2
 - @react-spectrum/utils@3.10.0
 - @react-spectrum/view@3.6.0
 - @react-spectrum/well@3.4.3
 - @react-stately/calendar@3.3.0
 - @react-stately/checkbox@3.4.3
 - @react-stately/collections@3.9.0
 - @react-stately/color@3.4.0
 - @react-stately/combobox@3.5.2
 - @react-stately/data@3.10.0
 - @react-stately/datepicker@3.5.0
 - @react-stately/dnd@3.2.2
 - @react-stately/flags@3.0.0-alpha.0
 - @react-stately/grid@3.7.0
 - @react-stately/layout@3.12.2
 - @react-stately/list@3.9.0
 - @react-stately/menu@3.5.3
 - @react-stately/numberfield@3.5.0
 - @react-stately/overlays@3.6.0
 - @react-stately/radio@3.8.2
 - @react-stately/searchfield@3.4.3
 - @react-stately/select@3.5.2
 - @react-stately/selection@3.13.2
 - @react-stately/slider@3.4.0
 - @react-stately/table@3.10.0
 - @react-stately/tabs@3.5.0
 - @react-stately/toast@3.0.0-beta.1
 - @react-stately/toggle@3.6.0
 - @react-stately/tooltip@3.4.2
 - @react-stately/tree@3.7.0
 - @react-stately/utils@3.7.0
 - @react-stately/virtualizer@3.6.0
 - @react-types/actionbar@3.1.0
 - @react-types/breadcrumbs@3.6.0
 - @react-types/calendar@3.3.0
 - @react-types/card@3.0.0-alpha.16
 - @react-types/contextualhelp@3.2.3
 - @react-types/datepicker@3.4.0
 - @react-types/dialog@3.5.3
 - @react-types/inlinealert@3.0.0-rc.0
 - @react-types/link@3.4.3
 - @react-types/list@3.2.4
 - @react-types/menu@3.9.2
 - @react-types/overlays@3.8.0
 - @react-types/table@3.7.0
 - @react-types/tooltip@3.4.2
 - @spectrum-icons/color@3.5.3
 - @spectrum-icons/express@3.0.0-alpha.7
 - @spectrum-icons/illustrations@3.6.3
 - @spectrum-icons/ui@3.5.3
 - @spectrum-icons/workflow@4.2.2
 - react-aria@3.26.0
 - react-aria-components@1.0.0-alpha.5
 - react-stately@3.24.0
```

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+* Add `DropZone` component to React Aria Components - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/4451)
+* Add `TagGroup` component to React Aria Components - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4661)
+* Fix file upload dialog in React Aria Components `DropZone` - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/4681)
+* Make React Aria Components overlay open props override `DialogTrigger` state - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4684)
+* Add layout and orientation props to React Aria Components `ListBox` - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4669)
+* Update React Aria Component `Calendar` docs to show 3 months in `visibleDuration` - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4709)
+
+####React Aria Components Tabs API changes
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:85]: need the space...

```suggestion
#### React Aria Components Tabs API changes
```

File: packages/dev/docs/pages/releases/2023-06-26.mdx [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: TBD
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:14]: need description here, and update the date to 27?

File: packages/dev/docs/pages/releases/2023-06-26.mdx (hunk 11 above)
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2023-06-26.mdx:85]:
```suggestion
#### React Aria Components Tabs API changes
```

**Review by devongovett — APPROVED** [review]

**Pull request merged**